### ISSUE TITLE
Block dropdown arrows are positioned incorrectly after dynamically disabling or reconfiguring `custom-block-shape`

### ISSUE THREAD

**Issue by DNin01** [ScratchAddons/ScratchAddons#6692]
### Description

https://github.com/ScratchAddons/ScratchAddons/assets/106490990/6a426672-1548-4142-9f17-87bf1f2af99e

How much it's offset depends on your settings.

This seems to persist until you reload. If I were to guess, I'd think something is only being changed the first time you run the addon, not on dynamic disables or setting changes.

### Steps to reproduce

1. Open the editor.
2. Enable `custom-block-shape`
3. Adjust the padding size, or if it has already been changed, disable the addon.

### Environment

Browser: Edge 117
Scratch Addons version: https://github.com/ScratchAddons/ScratchAddons/commit/79fc955912e7c0359a6bb5aa4f8baeb7790d9eab
Operating system: Windows 11

**Comment by WorldLanguages** [comment]
Can you check if this bug was also here in stable version (v1.34.1) and maybe a couple versions back (v1.28.0)?

**Comment by DNin01** [comment]
It happens on v1.34.1 stable and with TurboWarp's outdated addons.

<details><summary>I also gotta find a better way to go back to earlier versions. The only efficient way I currently know how to do it is by switching branches in GitHub Desktop, and that resulted in some undesirable after-effects.</summary>
<p>

Not knowing what I was doing and having limited options due to using GitHub Desktop, I decided to switch to a branch (not a tag) consisting of an earlier version of Scratch Addons and ended up with some sort of conflict when I tried to switch back to master (probably because I have so many changed files on my clone of this repo from testing, which had the occasional merge conflict marker after switching to another branch. I should start putting my changes on branches instead) and Scratch Addons didn't load after I tried, until I went back to the latest version after finally resolving the conflicts by hand. I use GitHub Desktop pretty much exclusively, which is probably also why.

</p>
</details>
